.circleci/config.templ.yml

```diff
@@ -984,7 +984,9 @@ jobs:
       - run_test:
           pattern: 'molten'
 
-  mysqlconnector:
+  mysql_integrations:
```

Comment:
@emmettbutler I renamed this suite to hold all `mysql` integrations, and of course I am not seeing this entire suite running now 🤦 . No idea why this is happening

Reply:
Tahir has fixed this issue